packages/chain-events/src/chains/aave/filters/labeler.ts

```diff
@@ -29,6 +29,7 @@ export const Label: LabelerFilter = (
         linkUrl: chainId
           ? `/${chainId}/proposal/onchainproposal/${data.id}`
           : null,
+        icon: 'councilProposal',
```

Comment:
This is just a placeholder icon, need design guidance for all chain events and their correlating icons.